### PR TITLE
Move the log file to the user's home directory

### PR DESCRIPTION
**Pull request by shtukas**
Solves: https://github.com/guardian/ssm-scala/issues/92

### CODE REVIEW

File: src/main/resources/logback.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <configuration>
     <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-        <file>/tmp/ssm.log</file>
+        <file>${user.home}/.ssm/ssm.log</file>
```
Comment (rtyley) [inline src/main/resources/logback.xml]: Sounds legit- does it create the `~/.ssm/` folder if it doesn't exist?

**Comment by sihil** [comment]
Out of interest, do we really need a log file?

**Comment by shtukas** [comment]
Um... I would think it's a relic of early stages of ssm.  @aug24 @adamnfish Thoughts ?

File: src/main/resources/logback.xml (hunk 1 above)
Comment (shtukas) [reply src/main/resources/logback.xml]: Yes, it does. I tested it :)

**Comment by adamnfish** [comment]
I don't think there's actually much logging happening at the moment. Logging is nice though, it was very helpful during the early development phase.

This was the plan
https://github.com/guardian/ssm-scala/issues/1

Obviously, I don't know how to do that...

**Comment by shtukas** [comment]
Thanks Adam. I am merging this one for the moment.

**Comment by sihil** [comment]
`-v` when you need it, gets output to stderr?

**Pull request merged**